modules/ingest-common/build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
   compileOnly project(':modules:lang-painless')
   compile project(':libs:elasticsearch-grok')
   compile project(':libs:elasticsearch-dissect')
+  compile "org.apache.logging.log4j:log4j-api:${versions.log4j}"
```

Comment:
This is already included as a compileOnly dep through server, so it shouldn't be needed here?

Reply:
thanks, I missed that this was already a dependency